hub/src/main/resources/application.properties

```diff
@@ -1,2 +1,5 @@
-logging.level.ch.prevo.open=DEBUG
-open.prevo.hub.config.file=classpath:nodes.yaml
\ No newline at end of file
+logging.level.root=WARN
+logging.level.ch.prevo.open=INFO
```

Comment:
I think we should use here debug level, since e.g. the log output that the matching is started is debug. So we either change the level there or here.

Reply:
Done - changed the level to 'info' (matching is done relatively seldomly)